### ISSUE TITLE
s10+ with Android 12 won't let me download

### ISSUE THREAD

**Issue by AndrewFaucette** [tejado/Authorizer#74]
I have a Samsung Galaxy S10+ running android 12 and when I go to download the app it says "This app isn't available for your device because it was made for an older version of Android." I know this isn't the recommended device but shouldn't I still be able to download the app? TIA

**Comment by tejado** [comment]
Hi, you try to download it over the Play Store, correct? I guess I have to publish there a new version.

**Comment by AndrewFaucette** [comment]
Yes, is there another way to download it other than the play store?

**Comment by tejado** [comment]
Yes, here:
https://github.com/tejado/Authorizer/releases

Look for the authorizer-release.apk. Latest release is v0.5.0

**Comment by AndrewFaucette** [comment]
I got it downloaded! I was wondering how you set up FIDO? I've never used it before

**Comment by harhitosw** [comment]
> I got it downloaded! I was wondering how you set up FIDO? I've never used it before

@AndrewFaucette Once you install the .apk of the Authorizer app you can use an existing or create .psafe3 file in the app itself. After this for FIDO based sign-ins you can connect BLE of mobile to BLE of your laptop and try websites like webauthn.io where you register credentials and authenticate using registered credentials. You need to choose "Security Key" while you register and authenticate. For more info on FIDO protocol and other stuff you can use this repository [herrjemand/awesome-webauthn](https://github.com/herrjemand/awesome-webauthn).

**Comment by tejado** [comment]
@AndrewFaucette 
Please also see the README of Authorizer. There is a video which demonstrates the usage: https://www.youtube.com/watch?v=8ai8X0W5sh4
